scripts/install.sh

```diff
@@ -20,9 +20,19 @@ if [ -z "$SRC_FILES" ]; then
     echo "Error: No .c files found in the src directory."
     exit 1
 fi
-# Compile the source files
-gcc -g -Wall -Wextra -I"$PROJECT_ROOT/src" $SRC_FILES -o "$OUTPUT_BINARY" -lcrypto
-#gcc -I"$PROJECT_ROOT/src" $SRC_FILES -o "$OUTPUT_BINARY" -lcrypto
+
+# Create log directory and file with proper permissions
+sudo mkdir -p /var/log
```

Comment:
directory should exist but this is okay we also could store our logs in a directory /var/log/pproc but putting logs in /var/log also fine